File: Python/Program/Keyboard.py
from Periferal import Periferal

class Keyboard(Periferal):
    def __init__(self, merk="", nama="", koneksi="", tipeMekanikal="", layout=""):
        super().__init__(merk, nama, koneksi)
        self.tipeMekanikal = tipeMekanikal
        self.layout = layout
        
    def setTipeMekanikal(self, tipeMekanikal):
        self.tipeMekanikal = tipeMekanikal
        
    def getTipeMekanikal(self):
        return self.tipeMekanikal
    
    def setLayout(self, layout):
        self.layout = layout
        
    def getLayout(self):
        return self.layout
    
    
        

File: Java/Program/Keyboard.java
public class Keyboard extends Periferal {
    private String tipeMekanikal;
    private String layout;

    public Keyboard(String merk, String nama, String koneksi)
    {
        super(merk, nama, koneksi);
        this.tipeMekanikal = "";
        this.layout = "";
    }

    public Keyboard(String merk, String nama, String koneksi, String tipeMekanikal, String layout) {
        super(merk, nama, koneksi);
        this.tipeMekanikal = tipeMekanikal;
        this.layout = layout;
    }

    public void setTipeMekanikal(String tipeMekanikal) {
        this.tipeMekanikal = tipeMekanikal;
    }

    public String getTipeMekanikal() {
        return this.tipeMekanikal;
    }

    public void setLayout(String layout) {
        this.layout = layout;
    }

    public String getLayout() {
        return this.layout;
    }
}